.github/workflows/codeql-analysis-jvm.yml

```diff
@@ -26,13 +26,13 @@ jobs:
         with:
           languages: 'java'
 
-      - name: Setup Gradle
+      - name: Run tests
         uses: gradle/gradle-build-action@v2
         with:
           gradle-version: wrapper
-
-      - name: Build
-        run: gradle clean assemble --no-daemon
+          arguments: clean assemble --no-daemon
+        env:
+          PACKAGE_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Out of interest - why does this need this? Is it for case-creator?

Reply:
yep and any private repo - the migration stuff too, there's no harm in passing and not using it, but the GITHUB_TOKEN is limited is time and scope if it gets leaked Vs using the other secret.